src/OpenTelemetry/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ Notes](../../RELEASENOTES.md).
 
 ## Unreleased
 
+* Promote overflow attribute from experimental to stable and removed the
```

Comment:
Either change "promote" to "promoted" or change "removed" to "remove" to keep things consistent. (I personally prefer the latter if we want to do it across the repo).

Reply:
Fixed that.